src/engraving/dom/instrtemplate.cpp

```diff
@@ -792,7 +804,46 @@ const InstrumentTemplate* combinedTemplateSearch(const String& mxmlId, const Str
                 bestMatchStrength = matchStrength;
                 if (bestMatchStrength - nameWeight == perfectMatchStrength && nameWeight > 0) {
                     return bestMatch; // stop looking for matches
+                } else {
+                    // Initialize list
+                    instTemplateScoredVector.clear();
+                    minLevenshteinDistance = std::numeric_limits<int>::max();
+                }
+            }
+            // We keep a list of every instrument with the current bestMatchStrength
+            if ((matchStrength == bestMatchStrength) && (matchStrength > 0)) {
+                InstrumentTemplateScored instTemplateScored;
+                instTemplateScored.instrumentTemplate = it;
+
+                // if the name is meaningless keep the longest distance
+                if ((name.isEmpty()) || (name == u"MusicXML Part") || (name == u"Staff")) {
+                    instTemplateScored.score = std::numeric_limits<int>::max();
+                } else {
+                    // We keep the lowest distance with trackName ...
+                    instTemplateScored.score = muse::strings::levenshteinDistance(StaffName(
+                                                                                      name).toString().toStdString(),
+                                                                                  it->trackName.toStdString());
```

Comment:
Looks like uncrustify made a mess here; perhaps there should only be one line break namely after `levenshteinDistance(`.

Reply:
OK